src/Adapter/Cart/CommandHandler/CreateEmptyCustomerCartHandler.php

```diff
@@ -75,7 +75,7 @@ private function createEmptyCustomerCart(Customer $customer): Cart
 
         $cart->id_shop = $customer->id_shop;
         $cart->id_lang = (int) Configuration::get('PS_LANG_DEFAULT');
-        $cart->id_currency = (int) Configuration::get('PS_CURRENCY_DEFAULT');
+        $cart->id_currency = Currency::getDefaultCurrencyId();
```

Comment:
`use Currency;` should be added at the top.

Reply:
probably why all the behats fails